apps/src/applab/applab.js

```diff
@@ -666,6 +666,9 @@ Applab.init = function(config) {
   studioApp.runButtonClick = this.runButtonClick.bind(this);
 
   Applab.channelId = config.channel;
+  if (config.level.testAssetPathPrefix) {
+    Applab.assetPathPrefix = config.level.testAssetPathPrefix;
+  }
```

Comment:
alternately, I could pass this to applabMain(config) from runLevel(), though I'm not sure how much better that is than using config.level.